src/cgame/cg_servercmds.cpp

```diff
@@ -844,6 +847,11 @@ static void CG_Say( const char *name, int clientNum, saymode_t mode, const char
 					location = va( " (%s^*)", s );
 				}
 			}
+
+			if ( health < 1 )
+			{
+				location = va( " %s", "^1*DEAD*^*" );
```

Comment:
Maybe:

```c++
location = va( " ^1*%s*^*", _( "DEAD") );
```

to make the string translatable?

Reply:
Done in f36a9638c5ae431898a27b1e5360fbc33cd990e9